frontend/src/certificate_verification.tsx

```diff
@@ -0,0 +1,266 @@
+            setCertNotFound(true);
+          } else {
+            throw new Error("Något gick fel.");
+          }
+        })
+        .catch((error) => {
+          userMessage(error.message, "error");
+        });
+    };
+
+    if (file) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        const binaryFile = event.target?.result
+          ? event.target.result
+          : undefined;
+        postCertificate(id, binaryFile);
+      };
+      reader.readAsArrayBuffer(file);
+    }
+  };
+
+  const getLatestCertificate = (id: string) => {
+    fetch(`/api/certificates/preview/${id}`, {
+      method: "GET",
+      credentials: "same-origin",
+      headers: {
+        Accept: "application/octet-stream",
+        "Content-Type": "application/json",
+      },
+    })
+      .then((res) => res.arrayBuffer())
+      .then((data) => {
+        setPreviewWidth(window.innerWidth * 0.5);
+        setPreviewUrl(data);
+      })
+      .catch((error) => {
+        userMessage(error.message, "error");
+      });
+  };
+
+  // This function is necessary to make the previw work.
+  // Built into the library.
+  function onDocumentLoadSuccess({ numPages }) {
+    setNumPages(numPages);
+  }
+
+  return (
+    <>
+      <h1>Verifiera certifikat</h1>
+      <p>
+        Ladda upp ett digitalt certifikat för att se att det är signerat och
+        finns i databasen.
+      </p>
+      <div className={style.fileInput}>
+        <input
+          id="inputFile"
+          type="file"
+          onChange={(event) =>
+            setFile(event.target.files ? event.target.files[0] : undefined)
+          }
+          style={{ display: "none" }}
+        />
+        <label htmlFor="inputFile" className={style.label}>
+          <Button variant="contained" color="primary" component="span">
+            Välj fil...
+          </Button>
+        </label>
+        {file ? <p>{file.name}</p> : <p>Ingen fil vald.</p>}
+      </div>
+      <div className={style.flexRow}>
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={() => {
+            setCertValid(false);
+            setCertOutdated(false);
+            setCertNotFound(false);
+            setShowPdf(false);
+            verifyCertificate(id);
+          }}
+          disabled={file ? false : true}
+        >
+          Verifiera
+        </Button>
+      </div>
+      {certValid ? (
+        <Box
+          border={3}
+          borderRadius={8}
+          borderColor="success.light"
+          className={style.responseBox}
+        >
+          <div className={style.boxTitle}>
+            <h2>Certifikatet är giltig!</h2>
+            <CheckCircle className={style.successIcon} />
+          </div>
+          <p>
+            Dokumentet innehåller den senaste informationen om{" "}
+            {individual?.name} och finns i vår databas.
+          </p>
+        </Box>
+      ) : certOutdated ? (
+        <Box
+          border={3}
```

Comment:
```suggestion
          border=3
```

As mentioned by Jon.